docs/areatrigger_teleport.md

```diff
@@ -8,16 +8,15 @@ Contains all the teleport triggers definition. This table is used to complete .d
 
 **Structure**
 
-| Field                   | Type         | Attributes | Key | Null | Default | Extra | Comment    |
-|-------------------------|--------------|------------|-----|------|---------|-------|------------|
-| [id][1]                 | mediumint(8) | unsigned   | PRI | NO   | 0       |       | Identifier |
-| [name][2]               | text         | signed     |     | YES  | NULL    |       |            |
-| [target_map][3]         | smallint(5)  | unsigned   |     | NO   | 0       |       |            |
-| [target_position_x][4]  | float        | signed     |     | NO   | 0       |       |            |
-| [target_position_y][5]  | float        | signed     |     | NO   | 0       |       |            |
-| [target_position_z][6]  | float        | signed     |     | NO   | 0       |       |            |
-| [target_orientation][7] | float        | signed     |     | NO   | 0       |       |            |
-| [VerifiedBuild][8]      | smallint(5)  | signed     |     | YES  | 0       |       |            |
+| Field                   | Type         | Attributes | Key | Null | Default | Extra | Comment |
+|-------------------------|--------------|------------|-----|------|---------|-------|---------|
+| [ID][1]                 | mediumint(8) | unsigned   | PRI | NO   | 0       |       |         |
+| [Name][2]               | text         |            | MUL | YES  |         |       |         |
```

Comment:
here it's NULL the default value 
![image](https://user-images.githubusercontent.com/519778/104585969-1fcac480-5665-11eb-8426-308755a6cc5b.png)

Reply:
It is not entirely clear to me, because it shows me differently, possibly because of the database client. Anyway, in the SQL script, it appears as not null. But well, if I'm making a mistake, let me know, to correct it.

```sql
CREATE TABLE `areatrigger_teleport` (
  `ID` mediumint(8) unsigned NOT NULL DEFAULT '0',
  `Name` text,
  `target_map` smallint(5) unsigned NOT NULL DEFAULT '0',
  `target_position_x` float NOT NULL DEFAULT '0',
  `target_position_y` float NOT NULL DEFAULT '0',
  `target_position_z` float NOT NULL DEFAULT '0',
  `target_orientation` float NOT NULL DEFAULT '0',
  PRIMARY KEY (`ID`),
  FULLTEXT KEY `name` (`Name`)
) ENGINE=MyISAM DEFAULT CHARSET=utf8 ROW_FORMAT=FIXED COMMENT='Trigger System'
```
 I get a little confused when it appears in parentheses. Because for the most part, I wasn't wearing any.

![areatrigger_teleport](https://user-images.githubusercontent.com/2810187/104596008-226fee80-5652-11eb-8c4b-d32d508f4ece.png)